query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -125,6 +126,10 @@ case class SeriesKeysByFilters(filters: Seq[ColumnFilter],
                                startMs: Long,
                                endMs: Long) extends MetadataQueryPlan
 
+object TsCardinalities {
+  val SHARD_KEY_LABELS = Seq("_ws_", "_ns_", "__name__")
```

Comment:
This is baking-in the labels/structure of the shard key prefix, but it's necessary to select/order the shard key label-value pairs from a query. Maybe we could add a shard key prefix spec to the config?